draft-ietf-quic-qpack.md

```diff
@@ -96,36 +96,37 @@ reintroduced by HPACK when the loss includes a HEADERS frame.
 ## Avoiding Head-of-Line Blocking in HTTP/QUIC {#overview-hol-avoidance}
 
 Because QUIC does not guarantee order between data on different streams, a
-reference might be received on a request stream to an entry which is not yet in
-the dynamic table. References are called "vulnerable" when they refer to entries
-whose receipt has not been confirmed by the peer, because loss or reordering can
-keep the reference from being immediately usable.  If the reference cannot be
-immediately processed on receipt, the stream is considered "blocked."
-
-The decoder can signal that it is willing to process vulnerable references by
-setting SETTINGS_QPACK_BLOCKED_STREAMS to a non-zero value.  In this case, the
-encoder can choose on a per-header-block basis whether to favor higher
-compression ratio (by permitting vulnerable references) or HoL resilience (by
-avoiding them).
-
-An encoder MUST NOT have header blocks which contain vulnerable references
-outstanding on more streams than the value of SETTINGS_QPACK_BLOCKED_STREAMS at
-any time. Note that the decoder's count of streams which are actually blocked
-will be less than or equal to the encoder's count of references which are
-vulnerable (potentially blocking).  If the decoder encounters more blocked
+header block might reference an entry in the dynamic table that has not yet been
+received.
+
+Each header block contains a Largest Reference (see {{absolute-index}}) which
+identifies the table state necessary for decoding. If the greatest absolute
+index in the dynamic table is less than the value of the Largest Reference, the
+stream is considered "blocked."  While blocked, header field data MUST remain in
+the blocked stream's flow control window.  When the Largest Reference is zero,
```

Comment:
I don't think that there is an interoperability requirement here.  Holding data outstanding is advice, not a requirement.